agent/cmd/serve.go

```diff
@@ -105,38 +94,16 @@ var serveCmd = &cobra.Command{
 
 		// Create the automation service
 		autoServer, err := server.NewServer(ctx, workspace, &server.Options{
-			StackName: Stack,
+			StackName: _stack,
 		})
 		if err != nil {
 			log.Fatalw("unable to make an automation server", zap.Error(err))
 			os.Exit(1)
 		}
-
-		// Configure the grpc server.
-		// Apply zap logging and use filters to reduce log verbosity as needed.
-		address := fmt.Sprintf("%s:%d", Host, Port)
+		address := fmt.Sprintf("%s:%d", _host, _port)
 		log.Infow("starting the RPC server", "address", address)
-		serverOpts := []grpc_zap.Option{
```

Comment:
I felt better with this setup code being in serve.go rather than the server package, because a) the setup code tends to  be parameterized by flags, b) there might be other RPC services to be mixed into the server, and c) code that watches signals and calls os.Exit should be in the cmd package.  Moving the code has made this PR harder to understand.